optuna/study/__init__.py

```diff
@@ -0,0 +1,24 @@
+from optuna._callbacks import MaxTrialsCallback
+from optuna.study._study_direction import StudyDirection
+from optuna.study._study_summary import StudySummary
+from optuna.study.study import BaseStudy
+from optuna.study.study import copy_study
+from optuna.study.study import create_study
+from optuna.study.study import delete_study
+from optuna.study.study import get_all_study_summaries
+from optuna.study.study import load_study
+from optuna.study.study import Study
+
+
+__all__ = [
+    "MaxTrialsCallback",
+    "StudyDirection",
+    "StudySummary",
+    "copy_study",
+    "create_study",
+    "delete_study",
+    "get_all_study_summaries",
+    "load_study",
+    "BaseStudy",
+    "Study",
+]
```

Comment:
Is it okay that this list and [the list in API reference](https://67342-122299416-gh.circle-artifacts.com/0/docs/build/html/reference/study.html) are different?

Reply:
Thanks for your comment. I agree with you to keep the consistency. I'll fix it.

Reply:
Uh. I've noticed that `BaseStudy` and `copy_study` are missed in the docs. I think `BaseStudy` is a base class of the `Study`, so it can be missed in the docs. However, `copy_study` should be added in the docs, but it should be done in another PR. Thanks for the head up!

Reply:
[Just FYI] I created https://github.com/optuna/optuna/pull/2737.

Reply:
Thank you!